engine/build.gradle

```diff
@@ -152,6 +154,9 @@ dependencies {
         println "*** Setting a CrashReporter binary dependency, not present as source"
         compile group: 'org.terasology.crashreporter', name: 'cr-terasology', version: '4.0.1'
     }
+
+    compile 'com.google.protobuf:protobuf-java:3.2.0'
+    protobuf files("src/main/protobuf/EntityData.proto", "src/main/protobuf/NetMessage.proto")
```

Comment:
What is this line for? I thought that the plugin is clever enough to pick up the *.proto files from `src/main/protobuf` automatically. If not, you should specify a new source set for that purpose.